lookbook/docs/patterns/18-aria-live.md.erb

```diff
@@ -0,0 +1,93 @@
+  <div id="assertive" aria-live="assertive" aria-atomic="true" class="hidden-for-sighted"></div>
+</live-region>
+```
+
+The `@primer/live-region-element` package provides methods to trigger announcements programmatically:
+
+```ts
+import { announce } from '@primer/live-region-element';
+
+announce('Example polite message', { politeness: 'polite' });
+```
+
+### Using Turbo Streams to Trigger ARIA Live Updates
+To use it in a Rails Turbo response, and send updates to the screen reader after any action, we create a new Turbo stream component that triggers an ARIA action:
+```ruby
+OpTurbo::StreamComponent.new(action: :aria, message: "Form submission successful!", type: "polite", role: "status", target: "nil")
+```
+- This creates a **Turbo Stream update** with:
+  - `action: :aria` → Calls the `aria` action defined in `aria.stream.action.ts`
+  - `message: "Form submission successful!"` → The text announced by the screen reader
+  - `type: "polite"` → Ensures the message is read when the user is idle
+  - `role: "status"` → Helps screen readers recognize this as a status update
+  - `target: "nil"` → Ensures the update isn't inserted into the DOM but is handled as an announcement
+
+#### How to use it?
+In controller:
+```ruby
+    def render_aria_update_message(message:, type:, role: "alert")
+      turbo_streams << OpTurbo::StreamComponent.new(action: :aria, message:, type:, role:, target: "nil").render_in(view_context)
+    end
+
+    def update
+      render_success_flash_message_via_turbo_stream(message: I18n.t(:notice_successful_update))
+      render_aria_update_message(message: I18n.t("work_package_relations_tab.relations.create_child_aria_live_message"),type: "polite",role: "alert")
+      respond_with_turbo_streams
+    end
+```
+When updating via Turbo Frame:
+```erb
+<%= content_tag("turbo-frame", id: "frame-id") do %>
+....
+  <%= render OpTurbo::StreamComponent.new(
+    action: :aria,
+    message: "Update!!",
+    type: "polite",
+    target: "nil"
+  ) %>
+<% end %>
+
+```
+
+### Turbo Stream Action for ARIA Live
+The `registerAriaStreamAction` method defines a custom Turbo Stream action called `aria`. When triggered, it announces a message (using ARIA live regions) to screen readers, with a tone of either `"polite"` or `"assertive"`, depending on the `type` attribute in the stream element.
+```ts
+import { StreamActions, StreamElement } from '@hotwired/turbo';
+import { announce } from '@primer/live-region-element';
+
+export function registerAriaStreamAction() {
+  StreamActions.aria = function dialogStreamAction(this:StreamElement) {
+      const message = this.getAttribute('message') ?? '';
+      const type = this.getAttribute('type') ?? 'polite';
+      if (type === 'assertive') {
+        void announce(message, {
+          politeness: 'assertive',
+        });
+      } else {
+        void announce(message, {
+          politeness: 'polite',
+          delayMs: 5000,
+        });
+      }
```

Comment:
I really don't understand what this paragraph and the method is about or what it is used for. The headline says "Turbo Stream Action for ARIA Live" but the previous headline is also for Turbo stream actions with headlines..